### PR TITLE
 TargetLabel's fileLabel allow opening file's containing path

### PR DESCRIPTION
**Pull request by dpallotti**
With double click and context menu action.

Bug: http://b/191121155#comment2

### CODE REVIEW

File: src/OrbitQt/DoubleClickableLabel.h [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+#include <QMouseEvent>
+#include <QWidget>
+
+namespace orbit_qt {
+
+class DoubleClickableLabel : public QLabel {
+  Q_OBJECT
+
+ public:
+  explicit DoubleClickableLabel(QWidget* parent = nullptr) : QLabel{parent} {}
```
Comment (beckerhe) [inline src/OrbitQt/DoubleClickableLabel.h:18]: Nit: `using QLabel::QLabel;`

File: src/OrbitQt/DoubleClickableLabelTest.cpp [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+#include "DoubleClickableLabel.h"
+
+namespace orbit_qt {
+
+TEST(DoubleClickableLabel, DoubleClickedSignalledOnDoubleClick) {
+  DoubleClickableLabel label;
+  bool double_clicked_signalled = false;
+  QObject::connect(&label, &DoubleClickableLabel::DoubleClicked,
+                   [&double_clicked_signalled] { double_clicked_signalled = true; });
```
Comment (beckerhe) [inline src/OrbitQt/DoubleClickableLabelTest.cpp:17]: Nit: For testing signals there is `QSignalSpy` (https://doc.qt.io/qt-5/qsignalspy.html). But the current solution is also fine with me.

**Review by beckerhe — APPROVED** [review]
Very nice. Only super minor things. Feel free to ignore all of them.

**Review by antonrohr — APPROVED** [review]
Very nice, thank you!

File: src/OrbitQt/DoubleClickableLabel.h (hunk 1 above)
Comment (dpallotti) [reply src/OrbitQt/DoubleClickableLabel.h]: It even crossed my mind. Done.

File: src/OrbitQt/DoubleClickableLabelTest.cpp (hunk 2 above)
Comment (dpallotti) [reply src/OrbitQt/DoubleClickableLabelTest.cpp]: It seems to work well for this case, done.

**Pull request merged**